FirebaseSessions/Tests/TestApp/AppQualityDevApp-iOS-Info.plist

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict/>
+</plist>
```

Comment:
Are these new files and pbxproj changes required or can we delete them?

Reply:
These are not new changes. I was doing some local testing and that would have caused this situation. Will revert it back.

Reply:
Done.